rotkehlchen/api/rest.py

```diff
@@ -3568,11 +3570,11 @@ def get_history_events(
                 group_by='event_identifier' if group_by_event_ids else None,
             )
             location = filter_query.location
-            chain_id = None
-            if location == Location.ETHEREUM:  # todo: this should prolly become a function
-                chain_id = ChainID.ETHEREUM
-            elif location == Location.OPTIMISM:
-                chain_id = ChainID.OPTIMISM
+            if location in EVM_LOCATIONS:
```

Comment:
`to_chain_id`'s documentation says "Dealing directly with ints since it's only used in one place". Since it is now used in more than one place, and here `ChainID` class is needed, maybe it will make sense to modify `to_chain_id` to return `ChainID` ?

Reply:
Is not used anymore in only one place and the integer value is the thing used, not the chainId instance. Also the comment states that is there to avoid the import issues so I will keep it. Change it to a single line assignation